### DISCUSSION TITLE
Java 8 Runtime Possibility

### DISCUSSION THREAD

**Discussion by javid90khan** [qualersoft/jmeter-gradle-plugin#35] [Q&A] [answered]
Hi!

i love the plugin. The build agent I am using only has java 8 runtime. I am wondering if your plugin definitely need java 11?

if it supports java8 it would be great in terms of backward compatibility. If not then I will do the struggle and get the agent upgraded.

thanks.

**Comment by mathze** [comment, marked as answer]
Hi @javid90khan,
never thought about it😁 at the time of writing the plugin I just used the java-version I had on my machine😝 
Will have a look at it.

**Reply by mathze** [reply]
@javid90khan I had a closer look at it and 'downgrading' to Java8 seams to be no problem.
I'v create #36 to address this.
I will take care of it the next days. Hope you can stand a few days longer. :)

**Reply by javid90khan** [reply]
Thanks a lot. Good point on keeping it inline with JMeter compatibility. really appreciate the quick response.

**Reply by mathze** [reply]
@javid90khan 
Version [2.2.2](https://github.com/qualersoft/jmeter-gradle-plugin/releases/tag/v2.2.2) is release
Don't wonder where 2.2.1 is had a bug in my versioning script :P

**Reply by javid90khan** [reply]
Lovely!
Thanks for this.
